schemas/styles/csl-choose.rnc

```diff
@@ -64,6 +65,40 @@ div {
       attribute variable {
         list { variables+ }
       }
+  
+  # Tests that require two attributes
+  condition.atts.complex = 
+    condition.atts.complex.dates
+    |
+    condition.atts.complex.general
+
+  # Dates
+  condition.atts.complex.dates =
+    ## specifies the variable to be tested
+    attribute tested { 
+      list { variables.dates+ }
+     },
+     (date-precision | date-range)+
+    
+  ## Tests the precision of a date variable
+  date-precision = attribute date-precision { 
+    list { ("year" | "month" | "day") }
```

Comment:
This says this is valid:

``` xml
<if date-precision="year month"/>
```

Reply:
Thanks. I have changed this according to your suggestion, and I've also added a docstring.